demo/component/LineChartSyncId.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from 'react';
+import { LineChart, Line, XAxis, YAxis, Tooltip } from 'recharts';
+
+const data1 = [
+  { name: 'Page A', uv: 400, pv: 2400, amt: 2400 },
+  { name: 'Page B', uv: 300, pv: 4567, amt: 2400 },
+  { name: 'Page C', uv: 300, pv: 1398, amt: 2400 },
+  { name: 'Page D', uv: 200, pv: 9800, amt: 2400 },
+  { name: 'Page E', uv: 278, pv: 3908, amt: 2400 },
+  { name: 'Page F', uv: 189, pv: 4800, amt: 2400 },
+];
+
+// Reversed of data1
+const data2 = [
```

Comment:
Just to avoid the possible confusion with the value being the value on the Y axis, I think it's worth having a different set of `uv` values